### ISSUE TITLE
[IxButton] Changing children of a button throws `NotFoundError`

### ISSUE THREAD

**Issue by robert-wettstaedt** [siemens/ix#503]
### What happened?

Changing or replacing the children of a button yields the following error:

```
NotFoundError

Node.removeChild: The node to be removed is not a child of this node

▼ 21 stack frames were expanded.
removeChild
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:11099:18

commitDeletionEffectsOnFiber
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24028:26

commitDeletionEffects
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:23976:33

recursivelyTraverseMutationEffects
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24259:30

commitMutationEffectsOnFiber
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24346:43

recursivelyTraverseMutationEffects
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24273:35

commitMutationEffectsOnFiber
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24332:43

recursivelyTraverseMutationEffects
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24273:35

commitMutationEffectsOnFiber
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24293:43

recursivelyTraverseMutationEffects
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24273:35

commitMutationEffectsOnFiber
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24293:43

recursivelyTraverseMutationEffects
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24273:35

commitMutationEffectsOnFiber
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24576:43

recursivelyTraverseMutationEffects
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24273:35

commitMutationEffectsOnFiber
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24432:43

commitMutationEffects
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:24243:31

commitRootImpl
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:26810:26

commitRoot
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:26682:19

performSyncWorkOnRoot
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:26117:13

flushSyncCallbacks
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:12042:22

ensureRootIsScheduled/<
https://0wz3rg.codesandbox.io/node_modules/react-dom/cjs/react-dom.development.js:25651:13

```

### What type of frontend frameware are you seeing the problem on?

React

### Which version of iX do you use?

v1.4.2

### Code to produce this issue.

```shell
https://codesandbox.io/s/changing-children-of-a-ixbutton-throws-notfounderror-9iq2zj
```

**Comment by danielleroux** [comment]
@robert-wettstaedt This is a known problem regarding react and stenciljs-webcomponents which dont use shadowDOM as the encapsulation. Its planned to migrate the components to shadowDOM components then the ugly workround with an addional child wrapper is not necessary.

https://github.com/siemens/ix/issues/138

**Comment by danielleroux** [comment]
Fix with #534
